### PR TITLE
pseudo distributed no logs error

### PR DESCRIPTION
**Pull request by haimeiguo**
logback.xml not added in the conf folder. 
now each node's logs are moved to node1/ node2/ node3/ folders
image

### CODE REVIEW

**Review by neuyilan — APPROVED** [review]
Many thanks.
BTW, does the lockback file[1] is needed? if not, we can delete it.
[1] https://github.com/apache/iotdb/blob/master/cluster/src/test/resources/logback.xml

**Comment by HTHou** [comment]
> BTW, does the lockback file[1] is needed? if not, we can delete it.
> [1] https://github.com/apache/iotdb/blob/master/cluster/src/test/resources/logback.xml

Is it for starting one node?

**Review by HTHou — APPROVED** [review]

**Comment by neuyilan** [comment]
> > BTW, does the lockback file[1] is needed? if not, we can delete it.
> > [1] https://github.com/apache/iotdb/blob/master/cluster/src/test/resources/logback.xml
> 
> Is it for starting one node by `nohup ./cluster/target/iotdb-cluster-0.13.0-SNAPSHOT/sbin/start-node.sh >/dev/null 2>&1 & `

I think if we only start one node cluster using the command `nohup ./cluster/target/iotdb-cluster-0.13.0-SNAPSHOT/sbin/start-node.sh >/dev/null 2>&1 & `, the logback in the `iotdb-cluster-0.13.0-SNAPSHOT/conf` folder  is used.

File: cluster/src/test/resources/node2conf/logback.xml [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+        <filter class="ch.qos.logback.classic.filter.LevelFilter">
+            <level>error</level>
+            <onMatch>ACCEPT</onMatch>
+            <onMismatch>DENY</onMismatch>
+        </filter>
+    </appender>
+    <appender class="ch.qos.logback.core.rolling.RollingFileAppender" name="FILEWARN">
+        <file>${IOTDB_HOME}/logs_node2/log_warn.log</file>
+        <rollingPolicy class="ch.qos.logback.core.rolling.SizeAndTimeBasedRollingPolicy">
+            <fileNamePattern>${IOTDB_HOME}/logs_node2/log-warn-%d{yyyyMMdd-HH}.%i.log.gz</fileNamePattern>
```
Comment (mychaow) [inline cluster/src/test/resources/node2conf/logback.xml:48]: why use logs_node2?

Comment (haimeiguo) [reply cluster/src/test/resources/node2conf/logback.xml]: fixed

**Pull request merged**